content/blog/gmailr-2-0-0/index.Rmd

```diff
@@ -0,0 +1,183 @@
+---
+output: hugodown::hugo_document
+
+slug: gmailr-2-0-0
+title: gmailr 2.0.0
+date: 2023-06-29
+author: Jennifer Bryan
+description: >
+    gmailr 2.0.0 streamlines the auth process and makes it easier to use gmailr
+    in a cloud or deployed context.
+
+photo:
+  url: https://unsplash.com/photos/VybzKEUMhbw
+  author: Hiroshi Kimura
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [gmailr, gargle]
+---
+
+We're chuffed to announce the release of [gmailr](https://gmailr.r-lib.org/) 2.0.0.
+gmailr exposes the [Gmail API](https://developers.google.com/gmail/api/guides) from R.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("gmailr")
+```
+
+The main goal of version 2.0.0 is to improve the ergonomics around auth.
+There is less need for fussy code around configuring an OAuth client and it's easier to use gmailr in a non-interactive or deployed setting.
+There is also a major advance in the process of replacing legacy functions with versions that have a `gm_` prefix.
+The legacy functions still exist, but are now hard deprecated.
+Finally, gmailr no longer re-exports `%>%`, the magrittr pipe, now that we have `|>` in base R.
+
+You can see a full list of changes in the [release notes](https://github.com/r-lib/gmailr/releases/tag/v2.0.0).
+
+```{r setup}
+library(gmailr)
+```
+
+😬 *Ouch! These name collisions are **exactly** why gmailr added a universal `gm_` prefix to all of its functions starting in v1.0.0. One day in the not-too-distant future we can remove the troublesome legacy functions.*
+
+## OAuth client
+
+The Gmail API is more challenging to wrap, in terms of auth, than the APIs for Sheets, Drive, or BigQuery.
```

Comment:
Any relevant recent blog posts to link to here?

Reply:
Link added later in the paragraph.